index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Hello Microverse</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles.css">
+</head>
```

Comment:
- [ OPTIONAL]  Your path to your styling file has a small issue, which is breaking styling to appear on your web page. You can use something like `  <link rel="stylesheet" href="styles.css">` or add the dot before your slash `./`